src/components/hand-controls.js

```diff
@@ -44,7 +48,12 @@ EVENTS[ANIMATIONS.point] = 'pointing';
  * @property {string} Hand mapping (`left`, `right`).
  */
 module.exports.Component = registerComponent('hand-controls', {
-  schema: {default: 'left'},
+  // schema: {default: 'left'},
+  schema: {
+    hand: { default: 'left' },
+    // toon, low or high
+    handModelStyle: { default: 'low' }
```

Comment:
We need the values in the schema `{default: ‘lowPoly’, oneOf: [‘lowPoly’, ‘highPoly’, ‘toon’]}`